src/sentry/issues/grouptype.py

```diff
@@ -174,11 +174,20 @@ class GroupType:
     notification_config: NotificationConfig = NotificationConfig()
     detector_handler: type[DetectorHandler] | None = None
     detector_validator: type[BaseGroupTypeDetectorValidator] | None = None
+    detector_config_schema: dict[str, Any] | None = None
 
     def __init_subclass__(cls: type[GroupType], **kwargs: Any) -> None:
         super().__init_subclass__(**kwargs)
         registry.add(cls)
 
+        from sentry.workflow_engine.registry import detector_config_schema_registry
+
+        detector_config_schema = cls.detector_config_schema or {}
+        if (
+            cls.slug not in detector_config_schema_registry.registrations
+        ):  # TODO(cathy): remove after updating getsentry test with patch
```

Comment:
getsentry tests are failing because one of the tests is not patching the new registry. will need to update that before removing this